target/linux/ath79/dts/qca9531_glinet_gl-ar750.dts

```diff
@@ -63,8 +63,10 @@
 	i2c {
 		compatible = "i2c-gpio";
 
-		sda-gpios = <&gpio 17 GPIO_ACTIVE_LOW>;
-		scl-gpios = <&gpio 16 GPIO_ACTIVE_LOW>;
+		sda-gpios = <&gpio  1 (GPIO_ACTIVE_HIGH | GPIO_OPEN_DRAIN)>;
+		scl-gpios = <&gpio 16 (GPIO_ACTIVE_HIGH | GPIO_OPEN_DRAIN)>;
```

Comment:
ath79 GPIO driver doesn't support open drain mode.

Reply:
I got the flags from here:
https://www.kernel.org/doc/Documentation/devicetree/bindings/i2c/i2c-gpio.txt

I suppose the bit-bang driver is controlling the pin direction, so normal output is fine. I'll remove the GPIO_OPEN_DRAIN flags and re-test.

Reply:
Done.

I2cdetect still sees all my devices.